x-pack/test/fleet_api_integration/apis/epm/custom_ingest_pipeline.ts

```diff
@@ -14,42 +14,29 @@ const TEST_INDEX = 'logs-log.log-test';
 
 const CUSTOM_PIPELINE = 'logs-log.log@custom';
 
-let pkgVersion: string;
-
 export default function (providerContext: FtrProviderContext) {
   const { getService } = providerContext;
   const supertest = getService('supertest');
   const es = getService('es');
   const esArchiver = getService('esArchiver');
-
+  const LOG_INTEGRATION_VERSION = '1.1.2';
```

Comment:
Maybe we could add a comment to explain why the version was hardcoded? I guess this will need to change in the future for the next release.

Reply:
Adding a TODO to that issue will probably make sense here https://github.com/elastic/kibana/issues/154243